hooks/001-extra-packages.chroot

```diff
@@ -81,7 +81,7 @@ Pin-Priority: 1002
 EOF
 
 # enable desktop-snappers PPA
-cat << \EOF > /etc/apt/trusted.gpg.d/desktop-snappers-core-desktop.asc
+cat << EOF > /etc/apt/trusted.gpg.d/desktop-snappers-core-desktop.asc
```

Comment:
Was the backslash here a mistake?

Reply:
The backslash is valid shell syntax, indicating that environment variables and other escapes sequences shouldn't be expanded in the here document.

It probably doesn't make a difference here, but I tend to prefer the backslash version if we expect to use it verbatim.

Reply:
I removed it because it was the only place where it was used, and also gave some trouble to the coloring algorithm in VSCode to determine the ending of the block.

Reply:
But maybe I should send a bug report to Microsoft :-D